Utilities/GUI/DualTabbedLayoutStuff/DualTabbedLayout.cs

```diff
@@ -696,7 +696,7 @@ private void window_KeyDown(object sender, KeyEventArgs e)
         {
             Window window = (Window)sender;
             DualTabbedLayoutItem item = (DualTabbedLayoutItem)window.Tag;
-            if (e.Key == Key.F4 && (KeyboardTools.IsCTRLDown() || KeyboardTools.IsALTDown()))
+            if ((e.Key == Key.F4 || e.Key == Key.W) && (KeyboardTools.IsCTRLDown() || KeyboardTools.IsALTDown()))

```

Comment:
Nitpick: this selected for Ctrl+F4 and Alt+F4, now also selects for Ctrl+W and Alt+W. Be a bit stricter only allowing Ctrl+W, not Alt+W?

Reply:
My bad, this was unintentional. Fixed.